packages/neth/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@near-wallet-selector/neth",
+  "version": "0.6.0",
```

Comment:
Version has to be the same as for the other packages.  
Description, author, and license can be removed.